smart-contracts/LimitUpdating.md

```diff
@@ -2,9 +2,11 @@
 
 If you are trying to whitelist and update token limits many addresses at once, you will need to use the bulkSetTokenLockBurnLimit.js script. If you need to only update the lock or burn limit and the token has already been whitelisted, use the setTokenLockBurnLimit script.
 
-To bulk update the whitelist and limits for each token, use bulkSetTokenLockBurnLimit.js like so:
+Before running the following script go to the data folder and open the `limitWhitelistUpdate.json`. Edit the address value to the address you want to whitelist. Then, update the limit to the desired limit you would like to allow a user to move through the bridge in a single tx. If you want to set this to the max allowable value in solidity, set the limit value to `115792089237316195423570985008687907853269984665640564039457584007913129639935` which is the UINT_MAX in solidity.
+
+Make sure the private key in your .env file is the operator address, ensure the infura id is set correctly as well. Get the bridgebank address and set it in the env var when running the script. To bulk update the whitelist and limits for each token, use bulkSetTokenLockBurnLimit.js like so:
```

Comment:
I think this and other env setup will be applicable to other kinds of future tasks too. Can we move this into a general pre-requisite section/document like Runbook.md with a setup section like "Setting up your environment for doing operations work"

Reply:
Done.